### DISCUSSION TITLE
Auto Delete Other Exported File

### DISCUSSION THREAD

**Discussion by roytoh** [KosmosisDire/obsidian-webpage-export#530] [General]
When i export files, some folder not export auto deleted.

Just realise need clear cache before export, else it will erase files not export and export selected files only.

**Comment by KosmosisDire** [comment]
Hi, no actually you are supposed to select ALL the files that you want to be in your export! NOT ONLY THE NEW FILES. This way it can manage all the files at once and it will only actually export ones that don't already exist or have been updated since the last export.

**Reply by roytoh** [reply]
I unable select all files as my total files 21.3GB. So i need split out to export.
